pkg/v1/tkg/web/src/app/shared/contextual-help/build-script/convert.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios');
+const showdown = require('showdown');
+const pathData = require('./docPath.json');
+
+
+const CWD = process.cwd();
+
+const saveFile = (filePathDest, content) => {
+    fs.writeFile(filePathDest, content, err => {
+        if (err) {
+            console.error(err);
+        }
+    });
+};
+const convertFile = (data) => {
```

Comment:
nitpick, please add missing line break between functions throughout this file

Reply:
added.